CHANGELOG.md

```diff
@@ -4,12 +4,58 @@
 
 ### API changes
 
+#### Added
+
+- added `dc_context_t*   dc_context_new_closed(const char* dbfile)`, `int  dc_context_open(dc_context_t *context, const char* passphrase)`, `int  dc_context_is_open(dc_context_t *context)` and `uint32_t  dc_accounts_add_closed_account(dc_accounts_t* accounts)` #2956
+
 - added `DC_MSG_WEBXDC`, `dc_send_webxdc_status_update()`,
   `dc_get_webxdc_status_updates()`, `dc_msg_get_webxdc_blob()`,
   `dc_msg_get_webxdc_info()`, and `DC_EVENT_WEBXDC_STATUS_UPDATE` #2826
+
+- Add `dc_msg_force_plaintext()` API for bots #2847
+
+- allow to remove quotes on drafts `dc_msg_set_quote(msg, NULL)` #2950
+
+#### Removed
+
 - Removed `mvbox_watch` option. #2906
   It is automatically enabled whenever `mvbox_move` is enabled.
+- Removed `inbox_watch` option #2922
 
+#### Deprecated
+- deprecated `os_name` in `dc_context_new(const char* os_name, const char* dbfile, const char* blobdir)`, now you should pass `NULL` or an empty string.
+
+### Added
+- Add imap table to keep track of message UIDs #2909
+- add hop_info (#2751)
+- Start making it possible to write to mailing lists (#2736)
+- sql: enable `auto_vacuum=INCREMENTAL` #2931
+- Synchronize Seen status across devices #2942
+- Add API to set the database password #2956
+- Add webXdc #2826
+
+### Changed
+- selfstatus now defaults to empty
+- `dc_preconfigure_keypair` now takes ascii armored keys instead of base64. #2862
+- Put removed member in Bcc instead of To in the message about removal
+#2864
+- do not change the draft's message-id on updates and sending (#2887)
+- Improve group updates #2889
+- Merge `mvbox_move` and `mvbox_watch` options. #2903
+- allow removing quotes on existing drafts (#2950)
+- update provider database (11 Jan 2022) #2959
+- allow timeout for internal configure tracker API #2967 (python)
+
+### Fixed
+- Fix: Make add_parts() not early-exit #2879
```

Comment:
```suggestion
- Fix: Make `add_parts()` not early-exit #2879
```

Reply:
also `hop_parts`, above, in general, at least all function names an everyrhing that contains an underscore should be backticked to not confuse markdown.